docs/gui.rst

```diff
@@ -276,6 +276,24 @@ A *event filter* is a list combined of *key*, *type* and *(type, key)* tuple, e.
         mouse_x, mouse_y = gui.get_cursor_pos()
 
 
+.. attribute:: gui.running
+
+    :parameter gui: (GUI)
+    :return: (bool) ``True`` if ``ti.GUI.EXIT`` event occurred, vice versa
+
+    For example:
+
+    ::
+
+        while gui.running:
+            if gui.get_event(ti.GUI.ESCAPE):
+                gui.running = False
```

Comment:
Do we automatically update `gui.running`, or should users update it in these lines?

Reply:
Oh it's `ti.GUI.ESCAPE`, not the close button.

Reply:
We do, the `gui.running` is a getter + setter. Thank for pointing out, we should make the doc more clear here.